profile_test.go

```diff
@@ -0,0 +1,281 @@
+package eat
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+const (
+	InvalidProfile = 100
+	InvalidURL     = "abcd"
+	EmptyURL       = "  "
+	InvalidOID     = "xxxx"
+	EmptyOID       = "  "
+)
+
+// TestProfileURL_Basic tests the basic setting of Profile as URL string
+func TestProfileURL_Basic(t *testing.T) {
```

Comment:
the naming convention for the test functions is `Test<ObjectUnderTest>_<MethodName>_<TestedFeature>`, so this should be `TestProfile_Get_Basic` or similar

Reply:
Corrected please check!